server/repository/migrations/postgres/2022-03-27T10-00_update_changelog_upsert_with_sync_function/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE FUNCTION update_changelog_upsert_with_sync()
```

Comment:
update_changelog_upsert_with_is_sync_update ?

That's pretty long thought, but `with_sync` is too ambiguous imo

Reply:
I think DB method works very nicely here, thanks